docs/yul.rst

```diff
@@ -7,15 +7,13 @@ Yul
 .. index:: ! assembly, ! asm, ! evmasm, ! yul, julia, iulia
 
 Yul (previously also called JULIA or IULIA) is an intermediate language that can
-compile to various different backends
-(EVM 1.0, EVM 1.5 and eWASM are planned).
-Because of that, it is designed to be a usable common denominator of all three
-platforms.
-It can already be used for "inline assembly" inside Solidity and
-future versions of the Solidity compiler will even use Yul as intermediate
-language. It should also be easy to build high-level optimizer stages for Yul.
-
-In its flavour of inline-assembly, Yul can be used as a language setting
+compile bytecode output for different backends.
```

Comment:
Is it really correct to say ``Yul can compile bytecode output``? Shouldn't it rather be ``Yul can *be compiled to* bytecode``?